.vscode/extensions.json

```diff
@@ -2,11 +2,7 @@
   // See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
   // Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
   // List of extensions which should be recommended for users of this workspace.
-  "recommendations": [
-    "dbaeumer.vscode-eslint",
-    "esbenp.prettier-vscode",
-    "orta.vscode-jest",
-  ],
+  "recommendations": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode", "orta.vscode-jest", "trezm.type-doc-vscode"],
```

Comment:
Ah I missed this one, I was trying to show typedoc warnings in VS code IDE rather than running `pnpm docs:validate` but this extension doesn't work. Removed in latest commit.